SpatialGDK/Source/SpatialGDKServices/Private/LocalDeploymentManager.cpp

```diff
@@ -485,16 +492,46 @@ bool FLocalDeploymentManager::GetServiceStatus()
 		bLocalDeploymentRunning = false;
 		return false;
 	}
-	else
+	else if (ServiceStatusResult.Contains(TEXT("Local API service is running")))
 	{
-		UE_LOG(LogSpatialDeploymentManager, Verbose, TEXT("Spatial service running."));
+		bSpatialServiceInProjectDirectory = IsServiceInCorrectDirectory(ServiceStatusResult);
 		bSpatialServiceRunning = true;
 		return true;
 	}
 
 	return false;
 }
 
+bool FLocalDeploymentManager::IsServiceInCorrectDirectory(FString ServiceStatusResult)
```

Comment:
I've checked FString::Split is a const functions, so would be nice if the argument is const FString&.